### PR TITLE
Working Docker setup with python3

### PR DESCRIPTION
**Pull request by allwyn-pradip**
These changes will allow docker to be setup and

### CODE REVIEW

**Comment by diegocepedaw** [comment]
@allwyn-pradip please make sure all the circleci tests are passing.

Looks like you have some flake8 errors

```
src/iris/webhooks/freshping.py:8:1: F401 'iris.webhooks.webhook.webhook' imported but unused
src/iris/webhooks/freshping.py:41:34: F821 undefined name 'basestring'
src/iris/webhooks/freshping.py:71:1: W293 blank line contains whitespace
Makefile:29: recipe for target 'flake8' failed
```

**Comment by allwyn-pradip** [comment]
update the conflicts

File: configs/config.dev.yaml [hunk 1]
```diff
@@ -67,13 +68,13 @@ user_session:
   sign_key: '123'
 
 supported_timezones:
+  - 'Asia/Kolkata'
```
Comment (diegocepedaw) [inline configs/config.dev.yaml:71]: small nit: can you change this ordering back to how it was previously so timezones are grouped by regions in the list

File: configs/config.dev.yaml [hunk 2]
```diff
@@ -132,7 +133,7 @@ sender:
   ## RPC access log
   access_log:
     maxBytes: 10485760
-    filename: './logs/sender_rpc.access.log'
+    filename: '/home/iris/var/log/sender_rpc.access.log'
```
Comment (diegocepedaw) [inline configs/config.dev.yaml:136]: do not change the default logpath

File: setup.py [hunk 3]
```diff
@@ -11,7 +11,7 @@
     long_description = fd.read()
 
 setuptools.setup(
-    name='irisapi',
+    name='iris',
```
Comment (diegocepedaw) [inline setup.py:14]: the name irisapi is tied to the pypi project, do not change it here: https://pypi.org/project/irisapi/

**Review by diegocepedaw — CHANGES_REQUESTED** [review]

File: configs/config.dev.yaml (hunk 2 above)
Comment (allwyn-pradip) [reply configs/config.dev.yaml]: changes done

File: setup.py (hunk 3 above)
Comment (allwyn-pradip) [reply setup.py]: changes done

File: configs/config.dev.yaml (hunk 1 above)
Comment (allwyn-pradip) [reply configs/config.dev.yaml]: changes done

**Pull request merged**